service-commands/src/commands/seed/UserCache.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs')
+
+const {
+  Constants
+} = require('../../utils')
+
+const {
+  SEED_CACHE_PATH
+} = Constants
+
+class UserCache {
```

Comment:
This started out as a `UserCache` but is now more of a `SeedCache` - hence the constant naming. But I stopped short of making it a full-on `SeedCache` because I wondered whether it made more sense to simply use the getters in the libs APIs, and also wanted to get more feedback before inadvertently duplicating a lot of state here in a weird data structure.

Reply:
not sure what it would entail if it was a full blown seed cache vs a user cache?

Reply:
I think it would allow us to randomize more actions if we kept a local cache of all tracks, playlists, social actions etc. But then at that point perhaps it doesn't make sense to create a duplicative local data store, and we should instead find a way to call the libs API getters from the seed CLI.